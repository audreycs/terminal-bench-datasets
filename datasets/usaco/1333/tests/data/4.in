10
147825 752503
960741 512108
83790 812992
328545 187606
182751 847169
818963 814953
706940 588194
981994 867047
248103 887086
501412 307158
