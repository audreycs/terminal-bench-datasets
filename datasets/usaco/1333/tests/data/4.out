88
68
62
156
76
446
98
58
68
80
